lib/repositories/products_repository.dart

```diff
@@ -9,13 +9,29 @@ class ProductsRepository {
 
   ProductsRepository(this.httpClient);
 
-  String get endpointUrl => '/products';
+  String get productsEndpointUrl => '/products';
 
   Future<List<Product>> getAllProducts() async {
-    final response = await httpClient.get(endpointUrl);
+    final response = await httpClient.get(productsEndpointUrl);
 
     return (response.data as List)
         .map<Product>((jsonProduct) => Product.fromJson(jsonProduct))
         .toList();
   }
+
+  // Category
+
+  String get categoriesEndpoint => '/categories';
```

Comment:
Locate all getters in one place - after constructor

Reply:
Maybe misunderstood the docs
https://github.com/VSydorko/flutter_guidelines/blob/feature/documentation/documentation/quidelines_documentation.md#methods-sorting
it's said that I need to separate entities in such case

Reply:
Yep, you right, but I'm talking about get methods, they should be located in one place
```dart

  String get productsEndpointUrl => '/products';
  String get categoriesEndpoint => '/categories';
```